var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -72,7 +72,7 @@ class Ascent(Package):
     # package dependencies
     ###########################################################################
 
-    depends_on("cmake@3.9.2:3.9.999", type='build')
+    depends_on("cmake@3.14.5", type='build')
```

Comment:
It doesn't build with newer versions? I think this should be:
```suggestion
    depends_on("cmake@3.14:", type='build')
```

Reply:
I will relax to allow newer vers